### DISCUSSION TITLE
There should be a release for the next beta now?

### DISCUSSION THREAD

**Discussion by karl-police** [Chatterino/chatterino2#2910] [Q&A] [answered]
I think the current Nightly Build seems good for beta release? Or not?

**Comment by zneix** [comment]
There's a bug (see #2822) which stops us from rolling out another release. Bad luck wanted that the animated emotes were rolled out just when we aren't quite ready to make another release.
There are nightly builds, so that you can have those newer features knowing that there might be bugs and it's entirely up to you if you want to install it.

**Reply by karl-police** [reply]
but beta? 🤔

**Reply by karl-police** [reply]
it's a beta release not final release

**Comment by jupjohn** [comment]
AFAIK @fourtf is preparing 2.3.3 and trying to find a build where that bug isn't present. We'll see how that goes.

**Comment by ALazyMeme** [comment, marked as answer]
It has been released.
